lib/location/Geocoding.ts

```diff
@@ -32,10 +33,11 @@ export class ExpoGeocoding implements Geocoding {
   }
 
   async reverseGeocode (location: Location) {
-    const results = await ExpoLocation.reverseGeocodeAsync({
-      ...location
-    })
-    return results as Placemark[]
+    return nullsToUndefined(
```

Comment:
Not sure if we want/need to use this function gratuitously

Reply:
I probably should add a note about this in the docs for it, but generally I agree. Its only real use-case imo is for converting types from 3rd party libraries that use null, however it's still a nice to have I think.